src/drawers/rectDrawer.ts

```diff
@@ -13,7 +13,7 @@ export module _Drawer {
 
     constructor(key: string, isVertical: boolean) {
       super(key);
-      this.svgElement("rect");
```

Comment:
We will probably want to revisit Drawers.  I think it turned from something that was simple to something that is complicated and blackboxed.